cmd/trade.go

```diff
@@ -308,49 +312,48 @@ func validateTrustlines(client *horizon.Client, botConfig *trader.BotConfig) {
 	}
 
 	if len(missingTrustlines) > 0 {
-		log.Println()
-		log.Fatalf("error: your trading account does not have the required trustlines: %v\n", missingTrustlines)
+		e := fmt.Errorf("error: your trading account does not have the required trustlines: %v", missingTrustlines)
+		logger.Fatal(l, e)
 	}
 }
 
 func deleteAllOffersAndExit(botConfig trader.BotConfig, client *horizon.Client, sdex *plugins.SDEX) {
-	log.Println()
-	log.Printf("deleting all offers and then exiting...\n")
+	l.Info("")
+	l.Infof("deleting all offers and then exiting...\n")
 
 	offers, e := utils.LoadAllOffers(botConfig.TradingAccount(), client)
 	if e != nil {
-		log.Println()
-		log.Fatal(e)
+		logger.Fatal(l, e)
 		return
 	}
 	sellingAOffers, buyingAOffers := utils.FilterOffers(offers, botConfig.AssetBase(), botConfig.AssetQuote())
 	allOffers := append(sellingAOffers, buyingAOffers...)
 
 	dOps := sdex.DeleteAllOffers(allOffers)
-	log.Printf("created %d operations to delete offers\n", len(dOps))
+	l.Infof("created %d operations to delete offers\n", len(dOps))
 
 	if len(dOps) > 0 {
 		e := sdex.SubmitOps(dOps, func(hash string, e error) {
 			if e != nil {
-				log.Println()
-				log.Fatal(e)
+				logger.Fatal(l, e)
 				return
 			}
-			log.Fatal("...deleted all offers, exiting")
+			e = fmt.Errorf("...deleted all offers, exiting")
+			logger.Fatal(l, e)
 		})
 		if e != nil {
-			log.Println()
-			log.Fatal(e)
+			logger.Fatal(l, e)
 			return
 		}
 
 		for {
 			sleepSeconds := 10
-			log.Printf("sleeping for %d seconds until our deletion is confirmed and we exit...\n", sleepSeconds)
+			l.Infof("sleeping for %d seconds until our deletion is confirmed and we exit...\n", sleepSeconds)
 			time.Sleep(time.Duration(sleepSeconds) * time.Second)
 		}
 	} else {
-		log.Fatal("...nothing to delete, exiting")
+		e := fmt.Errorf("...nothing to delete, exiting")
+		logger.Fatal(l, e)
```

Comment:
consolidate into one line here too (and anywhere else where it was missed up above too)